File: andrew035/PEOO/lde_3/Q3.java
import java.util.Scanner;
public class Q3{
    public static void main(String[]args){
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Informe o tamanho do vetor: ");
        int tamanho = s.nextInt();

        double[] n = new double[tamanho];

        for(int i = 0 ; i < tamanho ; i++){
            System.out.print("Informe o número: ");
            n[i] = s.nextDouble();
        }
        for(int i = 0 ; i < tamanho ; i++){
            if(n[i] % 2 != 0){
                System.out.println(n[i]);
            }
        }
    }
}

File: jairo015/fluxo/Q3.py
n1 = int(input("Informe N1: "))
n2 = int(input("Informe N2: "))
n3 = int(input("Informe N3: "))

if n1 > n2:
    if n1 > n3:
        print("O maior número é: " + str(n1))
    else:
        if n2 > n1:
            if n2 > n3:
                print("O maior número é: " + str(n2))
        else:
            if n3 > n1:
                if n3 > n2:
                    print("O maior número é:"  + str(n3))
